### PR TITLE
Update samshee

### PR DESCRIPTION
**Pull request by nschcolnicov**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/demultiplex/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/demultiplex _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by nschcolnicov** [comment]
Created this PR for testing the module, I will update it after the nf-core module has been merged into nf-core/modules

File: conf/modules.config [hunk 1]
```diff
@@ -234,6 +234,9 @@ process {
     }
 
     withName: SAMSHEE {
+        ext.json_schema_validator = { params.json_schema_validator ? "--schema '${params.json_schema_validator}'" : "" }
+        ext.name_schema_validator = { params.name_schema_validator ? "--schema '${params.name_schema_validator}'" : "" }
+        ext.v1                    = { params.v1_schema ? "--output-format sectioned" : "" }
```
Comment (maxulysse) [inline conf/modules.config:239]: I think we can put all that in a ext.args and not ressort to creating non standard ext

Comment (nschcolnicov) [reply conf/modules.config:239]: Yes, I agree, Im still working on it

Comment (nschcolnicov) [reply conf/modules.config:239]: Haha no worries! 😄 .Don't want you to waste your time reviewing something that might change regardless, but any help is always appreciated!

**Review by grst — APPROVED** [review]
The only downside currently seems that it's not possible to specify multiple samplesheets of any of the json/name/file types. 

However, I'm not sure if anyone will ever use that and there are workarounds... So let's not overengineer that for now.

**Pull request merged**